packaging/linux/fedora/sunshine.spec

```diff
@@ -0,0 +1,237 @@
+%global build_timestamp %(date +"%Y%m%d")
+%undefine _hardened_build
+
+Name:           sunshine
+Version:        2024.911.215654  # TODO: can we get this dynamically?
+Summary:        Sunshine is a self-hosted game stream host for Moonlight.
+Release:        1%{?dist}
+License:        GPLv3-only
+URL:            https://github.com/LizardByte/Sunshine
+Source0:        sunshine.sysusers  # TODO: what is this?
+Source1:        Linux-x86_64.tar.gz  # TODO: convert build-deps to cmake project and remove this
+
+BuildRequires:  boost-devel
+BuildRequires:  cmake
+BuildRequires:  gcc
+BuildRequires:  gcc-c++
+BuildRequires:  libayatana-appindicator3-devel
+BuildRequires:  libcap-devel
+BuildRequires:  libcurl-devel
+BuildRequires:  libdrm-devel
+BuildRequires:  libevdev-devel
+BuildRequires:  libgudev
+BuildRequires:  libnotify-devel
+BuildRequires:  libva-devel
+BuildRequires:  libvdpau-devel
+BuildRequires:  libX11-devel
+BuildRequires:  libxcb-devel
+BuildRequires:  libXcursor-devel
+BuildRequires:  libXfixes-devel
+BuildRequires:  libXi-devel
+BuildRequires:  libXinerama-devel
+BuildRequires:  libXrandr-devel
+BuildRequires:  libXtst-devel
+BuildRequires:  git
+BuildRequires:  mesa-libGL-devel
+BuildRequires:  miniupnpc-devel
+BuildRequires:  npm
+BuildRequires:  numactl-devel
+BuildRequires:  openssl-devel
+BuildRequires:  opus-devel
+BuildRequires:  pulseaudio-libs-devel
+BuildRequires:  rpm-build
+BuildRequires:  systemd-udev
+BuildRequires:  systemd-rpm-macros
+%{?sysusers_requires_compat}  # TODO: what is this?
+BuildRequires:  wget
+BuildRequires:  which
+
+# Conditional BuildRequires for cuda-gcc based on Fedora version and architecture
+%if 0%{?fedora} >= 40 && "%{_target_cpu}" == "x86_64"
+BuildRequires:  cuda-gcc-c++  # TODO: why not use this for all builds?
```

Comment:
Wasn't sure if there were any unexpected repercussions for using this package on aarch64 and did not have the ability to check for myself. very likely it can be used for all builds.